WooCommerce/Classes/ViewModels/MainTabViewModel.swift

```diff
@@ -22,8 +24,24 @@ final class MainTabViewModel {
 
     private var statusResultsController: ResultsController<StorageOrderStatus>?
 
-    init(storesManager: StoresManager = ServiceLocator.stores) {
+    private let featureFlagService: FeatureFlagService
+
+    /// Whether we should show the reviews badge on the hub menu tab.
+    /// Even if we set this value to true it might not be shown, as there might be other badge types with more priority
+    ///
+    @Published private var shouldShowReviewsBadgeOnHubMenuTab: Bool = false
+
+    /// Whether we should show the new feature badge on the hub menu tab.
+    /// Even if we set this value to true it might not be shown, as there might be other badge types with more priority
+    ///
+    @Published private var shouldShowNewFeatureBadgeOnHubMenuTab: Bool = false
```

Comment:
Should we change the `NewFeature` naming in different vars/methods for something more specific like `PaymentsInMenu` or similar?

Reply:
Actually, I wanted to make it generic enough, thus the new feature naming instead of `PaymentsInMenu`. That way, if we add another badge for another new button in menu, we do not have to change this implementation.

Reply:
Sounds good! Thanks for clarifying :)